subjects/function.json

```diff
@@ -0,0 +1,16 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
```

Comment:
what do `req` and `res` refer to? (or what are they short for?)
At first read-through it isnt immediately obvious to me as to what they represent. are they standard binding names used by azure?